draft-irtf-cfrg-aegis-aead.md

```diff
@@ -925,13 +938,536 @@ t = S3 ^ (LE64(ad_len_bits) || LE64(msg_len_bits))
 Repeat(7, Update(t))
 
 if tag_length == 16: # 128 bits
-  tag = S0 ^ S1 ^ S2 ^ S3 ^ S4 ^ S5
+    tag = S0 ^ S1 ^ S2 ^ S3 ^ S4 ^ S5
 else:                # 256 bits
-  tag = (S0 ^ S1 ^ S2) || (S3 ^ S4 ^ S5)
+    tag = (S0 ^ S1 ^ S2) || (S3 ^ S4 ^ S5)
 
 return tag
 ~~~
 
+# Parallel modes
+
+Some CPUs such as Intel and Intel-compatible CPUs with the VAES extensions include instructions to efficiently apply the AES round function to a vector of AES blocks.
+
+The AEGIS-128X and AEGIS-256X modes are designed to take advantage of these instructions. They share the same properties as the ciphers they are based on, but can be significantly faster on these platforms, even for short messages.
+
+AEGIS-128X and AEGIS-256X are parallel evaluations of multiple AEGIS-128L or AEGIS-256 instances with distinct initial states. On CPUs with wide vector registers, different states can be stored in different 128-bit lanes of the same vector register, allowing parallel updates using vector instructions.
+
+The modes are parameterized by the parallelism degree. With 256-bit registers, 2 parallel operations can be applied to 128-bit AES blocks. With 512-bit registers, the number of instances can be raised to 4.
```

Comment:
Having these as numbers helps to make the connection with `2X` and `4X`, but if it looks odd, I've nothing against `two` and `four`

Reply:
That's what I thought and makes sense. Perhaps we should leave it. The 2 looks odd to me compared to the 4 because it's mid-sentence, but they should either both be numbers or words.